src/PIL/ImageShow.py

```diff
@@ -241,20 +262,19 @@ class XDGViewer(UnixViewer):
     """
 
     def get_command_ex(self, file, **options):
-        command = executable = "xdg-open"
+        executable = "xdg-open"
+        command = f"xdg-open {quote(file)}"
         return command, executable
 
 
 class DisplayViewer(UnixViewer):
     """
     The ImageMagick ``display`` command.
-    This viewer supports the ``title`` parameter.
     """
 
-    def get_command_ex(self, file, title=None, **options):
-        command = executable = "display"
-        if title:
-            command += f" -name {quote(title)}"
```

Comment:
Could you explain why you think display doesn't need a title argument?

Reply:
Same thing as for xv, on top of it I think that the option is used incorrectly here. Please see the doc for X Windows: https://linux.die.net/man/7/x. I think that the **-title** option would be more relevant here.